profiles/fscloud/variables.tf

```diff
@@ -12,6 +12,12 @@ variable "name" {
   description = "Name of the Postgresql instance"
 }
 
+variable "existing_hpcs_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto service."
+  type        = string
+  default     = null
```

Comment:
why the default value of null? For FSCloud its required to have a hyper protect instance